lib/std/debug.zig

```diff
@@ -342,6 +342,14 @@ pub fn assert(ok: bool) void {
     if (!ok) unreachable; // assertion failure
 }
 
+/// This function is like `assert` but will only generate code in Debug mode.
+/// It is functionally equivalent to C's assert macro.
+pub fn assertWeak(ok: bool) void {
+    if (builtin.mode == .Debug) {
```

Comment:
```suggestion
    if (runtime_safety) {
```

Reply:
Debug only is intentional. No code should be generated in ReleaseSafe